src/libraries/Microsoft.Extensions.FileProviders.Physical/src/PhysicalFileProvider.cs

```diff
@@ -292,11 +292,11 @@ public IFileInfo GetFileInfo(string subpath)
         }
 
         /// <summary>
-        /// Enumerate a directory at the given path, if any.
+        /// Enumerates a directory at the given path, if any.
         /// </summary>
         /// <param name="subpath">A path under the root directory. Leading slashes are ignored.</param>
         /// <returns>
-        /// Contents of the directory. Caller must check <see cref="IDirectoryContents.Exists"/> property. <see cref="NotFoundDirectoryContents" /> if
+        /// The contents of the directory. Caller must check the <see cref="IDirectoryContents.Exists"/> property. Returns <see cref="NotFoundDirectoryContents" /> if
         /// <paramref name="subpath" /> is absolute, if the directory does not exist, or <paramref name="subpath" /> has invalid
         /// characters.
         /// </returns>
```

Comment:
I feel like the second sentence belongs more to remarks. What do you think?

```suggestion
        /// The contents of the directory.
        /// </returns>
        /// <remarks>
        /// <para>The caller must check the <see cref="IDirectoryContents.Exists"/> property.</para>
        /// <para>Returns <see cref="NotFoundDirectoryContents" /> if <paramref name="subpath" /> is absolute,
        /// if the directory does not exist, or <paramref name="subpath" /> has invalid characters.</para>
        /// </remarks>
```